node/src/service.rs

```diff
@@ -307,9 +308,13 @@ pub async fn start_parachain_node(
                 is_validator: parachain_config.role.is_authority(),
                 enable_http_requests: false,
                 custom_extensions: move |_| vec![],
-            })
-            .run(client.clone(), task_manager.spawn_handle())
-            .boxed(),
+            })?;
+        task_manager.spawn_handle().spawn(
+            "offchain-workers-runner",
+            "offchain-work",
+            offchain_workers
+                .run(client.clone(), task_manager.spawn_handle())
+                .boxed(),
```

Comment:
Do we need this? We don't have off-chain workers

Reply:
We had this before, I made it compatible with the newest polkadot. That's the difference - they're disabled anyways.